pkg/cmd/root.go

```diff
@@ -47,7 +47,21 @@ var (
 var rootCmd = &cobra.Command{
 	Use:   "nitric",
 	Short: "helper CLI for nitric applications",
-	Long:  ``,
+	Long: `Nitric - The fastest way to build serverless apps
+
+To begin working with Nitric, run the 'nitric new' command:
+
+    $ nitric new
+
+This will prompt you to create a new project for your cloud and language of choice.
+
+The most common commands from there are:
+
+    - nitric run   : Run a nitric stack locally for development or testing
```

Comment:
Run doesn't really run a "stack" it runs the project since stacks are project + deployment target. What about something like "run a nitric project locally..." or "Run nitric on a local stack..."?